### PR TITLE
Add qualcomm AI 100 chip card.

### PR DESCRIPTION
**Pull request by vaatsalya123**

### CODE REVIEW

File: hardware/Cloud_AI_100.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Cloud_AI_100:
+  name: "Cloud AI 100"
+
+  image_url: placeholder.png
+
+  tags:
+    - hardware
+    - custom-hardware
+    - edge
```
Comment (guillesanbri) [inline hardware/Cloud_AI_100.yaml:9]: seems like its targeted towards cloud servers and not edge? :thinking:

File: hardware/Cloud_AI_100.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Cloud_AI_100:
+  name: "Cloud AI 100"
+
+  image_url: placeholder.png
```
Comment (guillesanbri) [inline hardware/Cloud_AI_100.yaml:4]:
```suggestion
  image_url: qualcomm_ai100.png
```

File: hardware/Cloud_AI_100.yaml (hunk 1 above)
Comment (vaatsalya123) [reply hardware/Cloud_AI_100.yaml:9]: Cloud AI 100 is designed to be a leading solution for datacenters who increasingly rely on infrastructure at the edge-cloud. Essentially the chips is built for low power usage making it ideal for edge hardware vendors. Essentially this is the main reason why I added this tag.

Comment (guillesanbri) [reply hardware/Cloud_AI_100.yaml:9]: fair enough, I'll merge it then thanks!

**Pull request merged**